frontend/src/components/ConfirmForgotPassword.js

```diff
@@ -0,0 +1,150 @@
+import { withStyles } from '@material-ui/core/styles'
+import { Button, Grid, Paper, TextField, Typography } from '@material-ui/core'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { forgotPasswordSubmit, forgotPassword } from '../actions/auth'
+import { Redirect } from 'react-router-dom'
+import ErrorMessage from './ErrorMessage'
+
+const styles = (theme) => ({
+  root: {
+    background: 'linear-gradient(to right, #5e60ce, #5390d9)',
+    display: 'flex',
+    flexDirection: 'column',
+    minWidth: '100%',
+    minHeight: '100vh',
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  paper: {
+    minWidth: '340px',
+    padding: theme.spacing.unit * 2
+  },
+  input: {
+    width: '100%'
+  },
+  button: {
+    width: '100%',
+    marginTop: theme.spacing.unit
+  }
+})
+
+class ConfirmForgotPassword extends Component {
+  state = {
+    email: this.props.auth.email,
```

Comment:
What is the reason to initialise the state with the logged-in user? If they forgot their password, would they be logged in? 😄

Reply:
I initialised the state with email, because it is confirmed email and comes from reducer. And I need email to resend code to the user. I thought this is the right approach. If not do you have any suggestion for it.

Reply:
ah yes, that makes sense, thank you!